packages/e2e-tests/specs/editor/various/typewriter.test.js

```diff
@@ -21,6 +21,9 @@ describe( 'TypeWriter', () => {
 		// Create first block.
 		await page.keyboard.press( 'Enter' );
 
+		// Create second block.
```

Comment:
@ellatrix Not urgent, but when you have a moment I'd love your take on why this specific commit makes the tests pass. 

For context, this PR changes the **padding** of the editing canvas to be smaller. This leaves more room for themes that don't add this padding themselves, i.e. 2020 which is being used for these tests.

The tests failing were 

> TypeWriter › should maintain caret position
> TypeWriter › should maintain caret position after leaving last editable

I tested both of those using `npm run test-e2e packages/e2e-tests/specs/edi
tor/various/typewriter.test.js  -- --puppeteer-interactive`, passing on master but failing on this branch prior to this commit.

My **guess** is that with the padding as it was, there was enough room on the canvas for the blocks to fit without the typewriter effect setting in. This would explain why simply adding some enter-key presses fixed the test for me. 

But it does suggest that this particular test is very sensitive to things style properties like padding, margin, line-height and font size. For example if TwentyTwenty updates the editor style to add this padding back, or to change a block margin, this test will likely fail again. Is there a way to make it more resilient than just adding more blocks? Or conversely, is simply adding more blocks an okay way to fix this test?